src/keysim.js

```diff
@@ -219,6 +221,14 @@ export class Keyboard {
         if (events & KeyEvents.INPUT) {
           const textinputEvent = this.createEventFromKeystroke('textInput', keystroke, target);
           target.dispatchEvent(textinputEvent);
+          const inputEvent = this.createEventFromKeystroke('input', keystroke, target);
+          target.dispatchEvent(inputEvent);
+        }
+      }
+      if (keystroke.keyCode === US_ENGLISH_ACTION_KEYCODE_MAP.BACKSPACE) {
+        if (events & KeyEvents.INPUT) {
+          const inputEvent = this.createEventFromKeystroke('input', keystroke, target);
```

Comment:
Understood. In this case, should I check `target.value.length` or `target.selectionStart`?